database/sqlite/sqlite_functions.c

```diff
@@ -2254,3 +2263,216 @@ void sql_load_node_id(RRDHOST *host)
 
     return;
 };
+
+#define SQL_REPLICATION_SAVE_HOST_ENTRIES_RANGE \
+    "INSERT OR REPLACE INTO replication_host_entries_range(host_id, after, before) VALUES (?1, ?2, ?3);"
+
+int replication_save_host_entries_range(uuid_t *host_id, time_t After, time_t Before)
+{
+    sqlite3_stmt *res = NULL;
+    int rc, param = 0;
+
+    if (unlikely(!db_meta)) {
+        if (default_rrd_memory_mode != RRD_MEMORY_MODE_DBENGINE)
+            return 0;
+        error_report("Database has not been initialized");
+        return SQLITE_ERROR;
+    }
+
+    if (unlikely(!res)) {
+        rc = prepare_statement(db_meta, SQL_REPLICATION_SAVE_HOST_ENTRIES_RANGE, &res);
+        if (unlikely(rc != SQLITE_OK)) {
+            error_report("Failed to prepare statement to save host entries range, rc = %d", rc);
+            return SQLITE_ERROR;
+        }
+    }
+
+    param++;
+    rc = sqlite3_bind_blob(res, 1, host_id, sizeof(*host_id), SQLITE_STATIC);
+    if (unlikely(rc != SQLITE_OK))
+        goto bind_fail;
+
+    param++;
+    rc = sqlite3_bind_int(res, 2, After);
+    if (unlikely(rc != SQLITE_OK))
+        goto bind_fail;
+
+    param++;
+    rc = sqlite3_bind_int(res, 3, Before);
+    if (unlikely(rc != SQLITE_OK))
+        goto bind_fail;
+
+    rc = execute_insert(res);
+    if (unlikely(rc != SQLITE_DONE))
+        error_report("Failed to insert host entries range, rc = %d", rc);
+
+    error("GVD: saved host entries range to sqlite: <%ld, %ld>", After, Before);
```

Comment:
Why are you reporting an `error` here? Is it not better to have an `info`?

Reply:
Left over from when I was testing this manually. I've removed these log messages because they don't help at all to understand what's going on in the replication component. For debugging potential problems it's better to enable `log replication operations` and watch the output of the `api/v1/replication` endpoint.